content/articles/personality-test-app/index.md

```diff
@@ -0,0 +1,266 @@
+                   raise ValueError("Invalid input")
+           except ValueError as error:
+               print(error)
+           else:
+               if answer == 'A':
+                   count_of_a = count_of_a + 1
+               if answer == 'B':
+                   count_of_b = count_of_b + 1
+               count = count + 1
+```
+
+#### Compute the identities with if statement
+We use our variable counters and if statements to compute the personality identity.
+The sectioning of the questions is in four parts and five questions each, making a total of 20 questions. We compute the dominant personality identity at every section of the test.  
+
+```python
+                   if count == 5:
+                       if count_of_a > count_of_b:
+                           personality_dichotomy = personality_dichotomy + 'E '
+                       else:
+                           personality_dichotomy = personality_dichotomy + 'I '
+                   else:
+                       if count == 10:
+                           if count_of_a > count_of_b:
+                               personality_dichotomy = personality_dichotomy + 'S '
+                           else:
+                               personality_dichotomy = personality_dichotomy + 'N '
+                   else:
+                           if count == 15:
+                               if count_of_a > count_of_b:
+                                   personality_dichotomy = personality_dichotomy + 'T '
+                               else:
+                                   personality_dichotomy = personality_dichotomy + 'F '
+                   else:
+                       if count == 20:
+                               if count_of_a > count_of_b:
+                                   personality_dichotomy = personality_dichotomy + 'J '
+                               else:
+                                   personality_dichotomy = personality_dichotomy + 'P '
+ ```
+
+#### Call the display function. 
+
+ ```python 
+        display_personality_type (personality_dichotomy)        
+```
+
+#### Function for Exiting the Application
+This is a separate function that checks the decision of the user.
+The sys module here enables us to exit from the terminal if the user decides not to take the test.
+
+```python
+def exit_application():
+   print("Exiting application...")
+   sys.exit(0)
+```
+#### Homepage function
+This function is designed to mimic a landing page on the web. We only accept two options (1 or 2). We also check for wrong inputs using python exception handling.
+
+```python
+def main():
+   user_input = input("""
+   Welcome to the Meyers Briggs Personality Test
+   Press 1  to take test
+   Press 2 to exit application -> """)
+   try:
+       if not (user_input == "1" or user_input == "2"):
+           raise ValueError("Invalid input")
+   except ValueError as error:
+       print(error)
+   else:
+       switcher = {
+           "1": run,
```

Comment:
You do not have a function called `run` in your application. Meaning that your application cannot run since the only viable option is exiting the application.